site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,522 @@
+        <li>
+          Organize the navigation items hierarchically, highlighting the most important elements and grouping similar
+          items together.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Customize responsibly.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          When customizing the navigation pattern, ensure that the changes do not negatively impact usability,
+          accessibility, or consistency.
+        </li>
+      </ul>
+    </li>
+  </ul>
+</div>
+
+<div
+  class="guideline-dont"
+  style="background: var(--color-error-light); padding: var(--spacing-s); margin-bottom: var(--spacing-layout-2-xs); max-width:600px;"
+>
+  <div class="guideline-dont-label" style="color:var(--color-error); margin-bottom:var(--spacing-s);">
+    <IconError style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle' }} size="s" />
+    <strong>Don't</strong>
+  </div>
+  <ul style="margin-bottom:0;">
+    <li>
+      <strong>Don't sacrifice clarity for style.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          While it's essential to maintain a visually appealing design, prioritize clarity and usability when
+          implementing the navigation pattern.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Don't disregard performance.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Optimize the navigation pattern for performance by reducing file sizes, minimizing HTTP requests, and using
+          efficient code.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Don't neglect mobile users.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Do not assume all users are on desktop devices; ensure the navigation pattern works well on mobile devices and
+          touchscreens.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Don't ignore feedback.</strong>
+      <ul style="margin-top:0;">
+        <li>Do not disregard user feedback or testing results when refining and improving the navigation pattern.</li>
+      </ul>
+    </li>
+    <li>
+      <strong>Don't overcomplicate.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Avoid adding unnecessary elements, features, or interactions to the navigation pattern, which may confuse or
+          overwhelm users.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Don't hardcode content.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Avoid hardcoding content or URLs in the navigation pattern, as this may make future updates more difficult.
+        </li>
+      </ul>
+    </li>
+  </ul>
+</div>
+
+---
+
+## Recommended order of navigation components
+
+When implementing a navigation pattern, it's essential to consider the order of navigation components to ensure a logical, intuitive, and user-friendly experience.
+
+<Image
+  src="/images/patterns/navigation/examples/navigation-pattern-component-order.png"
```

Comment:
Should we have another version as well without hero but with side navigation?

Reply:
Good idea!